b2sdk/b2http.py

```diff
@@ -298,10 +384,13 @@ def post_json_return_json(self, url, headers, params, try_count: int = TRY_COUNT
         # to indicate the timeouts should be designed.
         timeout = self.TIMEOUT_FOR_COPY
 
-        data = io.BytesIO(json.dumps(params).encode())
+        data = json.dumps(params).encode()
         return self.post_content_return_json(
             url,
-            headers,
+            {
+                **headers,
+                'Content-Type': 'application/json',
+            },
             data,
```

Comment:
Can't it be just 
```python
headers,
json=data,
```
?

Reply:
I would have to change everything below to support it, can't see how would that be worth the trouble

Reply:
My bad, for some reason I confused it with `requests.request(...)`. Brain glitch. You are right.